runtime/jcl/common/mgmtthread.c

```diff
@@ -1115,34 +1104,33 @@ handlerMonitorWaited(J9HookInterface** hook, UDATA eventNum, void* eventData, vo
  * @pre must not have VM access
  * @todo When should these references be deleted?
  * @param[in] env
- * @return error status
+ * @param[in] jcls The Class (static method).
```

Comment:
Please remove the trailing period (`.`) for consistency with other `@param` elements.

Reply:
fixed.